tests/conftest.py

```diff
@@ -29,5 +29,30 @@ def test_adata(test_counts, test_metadata):
 
 
 @pytest.fixture
-def statsmodels_stub(test_adata):
-    return StatsmodelsDE(adata=test_adata, design="~condition")
+def test_adata_minimal():
+    obs = pd.DataFrame(
+        [
+            ["A", "D1", "X", 0.2],
+            ["B", "D1", "X", 0.7],
+            ["A", "D2", "Y", 1.6],
+            ["B", "D2", "Y", 42],
+            ["A", "D3", "Y", 212],
+            ["B", "D3", "Y", 6023],
+        ],
+        columns=["condition", "donor", "other", "cont"],
```

Comment:
```suggestion
        columns=["condition", "donor", "other", "continuous"],
```

I have to admit, it took me too many second to realize the purpose of this column